guides/development/hack_on_the_trento.md

```diff
@@ -1,82 +1,111 @@
-# Hack on the trento
-
-- [Requirements](#requirements)
-- [Install dependencies](#install-dependencies)
-- [Development environment](#development-environment)
-- [Setup trento](#setup-trento)
-- [Start trento in the repl](#start-trento-in-the-repl)
-- [Environment Variables](#environment-variables)
-- [Scenario loading with photofinish](#scenario-loading-with-photofinish)
+# Hack on the Trento Web
 
 ## Requirements
 
-In order to run the trento web, you need
+In order to run the Trento Web application, you need to have the following software installed on your system:
```

Comment:
Not a big fan of the `you` person. We should try to avoid it in my opinon (and suggested by docs team members). Example: `To run the Trento Web application install the next software`.;
But well, seeing that it is fully used all around the doc, it is hard to change I guess...

Reply:
A good point, let's see maybe i can do it better :D